ocs_ci/ocs/ui/base_ui.py

```diff
@@ -40,27 +41,30 @@ def __init__(self, driver):
         self.screenshots_folder = os.path.join(
             os.path.expanduser(ocsci_config.RUN["log_dir"]),
             f"screenshots_ui_{ocsci_config.RUN['run_id']}",
+            os.environ.get("PYTEST_CURRENT_TEST").split(":")[-1].split(" ")[0],
         )
         if not os.path.isdir(self.screenshots_folder):
-            os.mkdir(self.screenshots_folder)
+            Path(self.screenshots_folder).mkdir(parents=True, exist_ok=True)
         logger.info(f"screenshots pictures:{self.screenshots_folder}")
         if config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM:
             self.storage_class = "thin_sc"
         elif config.ENV_DATA["platform"].lower() == constants.AWS_PLATFORM:
             self.storage_class = "gp2_sc"
 
-    def do_click(self, locator, timeout=30):
+    def do_click(self, locator, timeout=30, enable_screenshot=True):
```

Comment:
I'm thinking out load...Maybe the default for enable_screenshot should be false and when we do want we should send true. I think that in most cases except if we catch some timeout exception it should be false. We need to strive to have less screenshot taken. WDYT

Reply:
@shyRozen @MeridianExplorer I changed the enable_screenshot to False by default..